corgidrp/data.py

```diff
@@ -742,17 +945,56 @@ class DetectorParams(Image):
         'kgain' : 8.7,
         'fwc_pp' : 90000.,
         'fwc_em' : 100000.,
-        'rowreadtime' : 223.5e-6 # seconds
+        'rowreadtime' : 223.5e-6, # seconds
+        # number of EM gain register stages
+        'Nem': 604,
+        # slice of rows that are used for telemetry
+        'telem_rows_start': -1,
+        'telem_rows_end': None, #goes to the end, in other words
+        # pixel full well (e-)
+        'fwc': 90000,
+        # serial full well (e-) in EM gain register in EXCAM EMCCD
+        'fwc_em': 100000,
+        # cosmic ray hit rate (hits/m**2/sec)
+        'X': 5.0e+04,
+        # pixel area (m**2/pixel)
+        'a': 1.69e-10,
+        # Maximum allowable EM gain
+        'gmax': 8000.0,
+        # tolerance in exposure time calculator
+        'delta_constr': 1.0e-4,
+        # Overhead time, in seconds, for each collected frame.  Used to compute
+        # total wall-clock time for data collection
+        'overhead': 3,
+        # Maximum allowed electrons/pixel/frame for photon counting
+        'pc_ecount_max': 0.1,
+        # number of read noise standard deviations at which to set the
+        # photon-counting threshold
+        'T_factor': 5,
+        ######################
+        # These below are ultimately sourced from calibration files
```

Comment:
I don't think I understand this comment. Are these parameters redundant?

Reply:
I mean that the parameters below come from calibration activities.  So if the plan is to exclusively draw calibration numbers from the corresponding calibration class instances in data.py, then yes, these are redundant.

Reply:
I guess from the comment I just didn't understand what calibration files you're talking about here. If they're derived from other existing calibration files in the corgi-DRP then I don't think we want them here too, right?